proto/api.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
```

Comment:
This project is shared between all teams so let's rename this file to `network-api.proto`

Reply:
You're right. Done !